templates/contribute/git.md

```diff
@@ -0,0 +1,324 @@
+# Git Guide for Mathlib4 Contributors
+
+This guide is designed for mathematicians who are new to git but want to contribute to mathlib4. We'll walk through the essential git workflows step by step.
```

Comment:
```suggestion
This guide is designed for mathematicians who are new to git but want to contribute to the mathlib4 library.
Contributions are made via pull requests. We'll walk through the essential workflows step by step.
Note that there are many other guides on the web describing how to contribute to open-source projects
via pull requests.
```
I'm just wondering whether we should introduce the key phrase "pull request" early because this is what people can search for.